services/src/main/java/org/fao/geonet/api/processing/report/ErrorReport.java

```diff
@@ -24,15 +24,11 @@
 package org.fao.geonet.api.processing.report;
 
 import org.fao.geonet.Util;
-import org.opengis.annotation.XmlElement;
 
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
 
-/**
- * Created by francois on 20/05/16.
- */
-@XmlElement(value = "error")
```

Comment:
@fxprunayre I tested the report and looks fine, but I'm not sure why this is removed?

Reply:
@josegar74 when I restore that as ``@XmlElement(name="error")`` it produces an error:

> '@XmlElement' not applicable to type

Reply:
Ok, my concern was that instances of this class were serialised with the root element `error` and now use a different name due to this change.

I tested a batch editing report and looks ok, but would be good to have some confirmation from @fxprunayre

Reply:
> I tested a batch editing report and looks ok, but would be good to have some confirmation from @fxprunayre

I tested it https://github.com/geonetwork/core-geonetwork/pull/7416#issuecomment-1754434642 and it was ok. It was wrong usage of the annotation.